cloudevents/src/v1_0/tests.rs

```diff
@@ -0,0 +1,140 @@
+use super::CloudEventBuilder;
```

Comment:
not the your test file is not rust compliant...

"You’ll put unit tests in the src directory in each file with the code that they’re testing. The convention is to create a module named tests in each file to contain the test functions and to annotate the module with cfg(test)." https://doc.rust-lang.org/book/ch11-03-test-organization.html#unit-tests